sysutils/terragrunt/Portfile

```diff
@@ -15,10 +15,23 @@ maintainers         {mjrc.nl:macports @mjrc} \
 # *NOTE* Remember to update `latestVersion` on a version upgrade.
 set latestVersion       terragrunt-0.43
```

Comment:
Looks like you should update this ^

Reply:
You are right! I've pushed a fix! Thanks for noticing it